source/deploy/server/kubernetes/deploy-k8s-aks.rst

```diff
@@ -48,32 +46,30 @@ The installation process includes deploying Mattermost and updating the server.
   .. image:: /_static/images/azure/aks-cluster.png
     :alt: An example of the Azure AKS cluster setup screen.
 
-4. Fill in the details for your PostgreSQL database.
+4. Fill in the details for your PostgreSQL database. Ensure the user specified has full access.
 
   .. image:: /_static/images/azure/postgreSQL.png
     :alt: An example of the Azure AKS Database setup screen.
 
-.. note::
-
-  - Connectivity should be already in place between the AKS cluster and the database.
-  - Database should already exist and the user specified must have full access.
 
-5. Adjust deployment details.
+5. Specify Deployment Details including Deployment Name and Deployment Size. Click the checkbox to Deploy Minio, a required utility for this installation that will provide filestore functionality for your Mattermost instance. 
 
   .. image:: /_static/images/azure/deployment-details.png
     :alt: An example of the Azure AKS Deployment Details setup screen.
 
-.. note:: 
-  You can define a Deployment size to automatically adjust the installation. A valid Mattermost license is required for deployments of more than 100 users.
-
 6. Configure Mattermost installation hostname and Ingress details. The AGIC add-on is used in the following example to show the ingress annotations required.
+  a. You can use any pre-installed Ingress Controller in your cluster as long as it supports Kubernetes Ingress and TLS termination.
 
   .. code-block:: yaml
 
     kubernetes.io/ingress.class: azure/application-gateway
     appgw.ingress.kubernetes.io/ssl-redirect: "true"
   
```

Comment:
Formatting on the note addition looks a bit off 
![image](https://github.com/user-attachments/assets/57144798-e419-4c19-96ea-726bb3a17f1a). I'm not sure I'm familiar enough with RST to suggest the proper fix here. @cwarnermm any ideas?